### PR TITLE
cs/cas-165 Keyword-Merging Functionality

### PR DESCRIPTION
**Pull request by CorySanin**
Integrates [collective/Products.PloneKeywordManager](https://github.com/collective/Products.PloneKeywordManager) into Castle, allowing users to replace and/or merge keywords from a handy control panel item. PloneKeywordManager's interface has been simplified by overriding the view's template with jbot, making it even easier to use.

### CODE REVIEW

**Comment by tkimnguyen** [comment]
- on the keyword manager control panel, `No content in this site has any keywords assigned yet, so there's nothing to manage at this time. (To assign keywords to a piece of content, use the item's "Categorization" tab while editing.)` is not quite correct for Castle: you have to click the PROPERTIES button at top left before you can see the Categorization tab.

**Comment by CorySanin** [comment]
All I'm missing at this point is tests

**Comment by CorySanin** [comment]
Tests are in. Should be ready to merge

File: castle/cms/tests/test_keywordmanager.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from Products.PloneKeywordManager.tests.test_controlpanel import ControlPanelTestCase
+from Products.PloneKeywordManager.tests.test_dexterity import DexterityContentTestCase
+from Products.PloneKeywordManager.tests.test_non_ascii_keywords import NonAsciiKeywordsTestCase
+from Products.PloneKeywordManager.tests.test_setup import InstallTestCase
+from castle.cms.testing import CASTLE_PLONE_INTEGRATION_TESTING
+
+class ControlPanel(ControlPanelTestCase):
```
Comment (vangheem) [inline castle/cms/tests/test_keywordmanager.py]: What is going on with this file? Should we remove this if we're not going to have any tests related to it?

Comment (CorySanin) [reply castle/cms/tests/test_keywordmanager.py]: The tests are inherited from the tests in Products.PloneKeywordManager

**Comment by CorySanin** [comment]
fyi, that pending Travis build has completed successfully. No idea why GitHub thinks it didn't finish.

**Comment by CorySanin** [comment]
I'll have to do a little cleanup when I get the chance 😛

**Comment by vangheem** [comment]
@CorySanin I think I have the flake8 issues fixed. Are there more things to cleanup?

**Comment by CorySanin** [comment]
Don't think so

**Comment by vangheem** [comment]
cool, I'll merge once travis is green!

**Pull request merged**